exporter/humioexporter/README.md

```diff
@@ -1,7 +1,12 @@
 # Humio Exporter
-Exports data to Humio using JSON over the HTTP [Ingest API](https://docs.humio.com/reference/api/ingest/).
 
-Supported pipeline types: traces (with metrics and logs to follow soon)
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
+
+Exports data to Humio using JSON over the HTTP [Ingest API](https://docs.humio.com/reference/api/ingest/).
 
 > :construction: This exporter is currently intended for evaluation purposes only! It has yet to be enabled in the build.
```

Comment:
Should this be removed?  The exporter is in the contrib distribution

Reply:
Yes, we forgot to remove that comment way back when the exporter was enabled. My former colleague and I have evaluated it internally (we are not affiliated with Humio), so we do consider it sufficiently stable for beta.

Reply:
ok ,I will remove the comment.